src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -2239,14 +2241,21 @@ void AuraEffect::HandleFeignDeath(AuraApplication const* aurApp, uint8 mode, boo
                 }
             }
         }
-        target->CombatStop();
+
+        if (target->GetMap()->IsDungeon()) // feign death does not remove combat in dungeons
```

Comment:
No, because feign death adds `UNIT_STATE_DEAD`, which is checked in `ThreatReference::selectOnlineState` and sets the threat ref to offline. It only maintains the actual combat state, e.g. you don't get potion CDs back and such.